src/prefect/deployments.py

```diff
@@ -1,51 +1,5 @@
 """
 Objects for specifying deployments and utilities for loading flows from deployments.
-
```

Comment:
Yup

Reply:
@chrisguidry I would expect that after adding a version on a deployment, users might leverage that for versioning their deployed flows (e.g. with the version being GITHUB_SHA) and will use the version set on the flow decorator only to version flows triggered without deployments (e.g., triggered locally or e.g. from Amazon Lambda function)

afaik, we currently don't expose this version anywhere in the UI or CLI, so currently, we only use this attribute internally without exposing it in any way to users